http/generators.http

```diff
@@ -7,4 +7,17 @@ GET {{host}}/{{generator}}/lot-pdf-result
 Content-Type: application/json
 
 ### generate laptopItems for one already exist lotPDFResult
-GET {{host}}/{{generator}}/laptop-item/{{lotPDFResult}}
\ No newline at end of file
+GET {{host}}/{{generator}}/laptop-item/{{lotPDFResult}}
+
+### receive all lots from prozorro api
+GET {{host}}/extract-lotid-all
+Content-Type: application/json
+
+### receive all lots information from prozorro api
+GET {{host}}/extract-lot-information-all
+Content-Type: application/json
+
+### receive one lot from prozorro api
+GET {{host}}/:lotId
+//lotId=b03c713a16a34d3c94cce7010d9081a4
```

Comment:
is it commented or a way how to set variables?

Reply:
It's example of lotId, for test.